.travis.yml

```diff
@@ -8,15 +8,17 @@ php:
   - 7.0
 env:
   - SYMFONY_CONSOLE_VERSION=2.8.*
-  - SYMFONY_CONSOLE_VERSION=3.0.*
-  - SYMFONY_CONSOLE_VERSION=3.1.*
-  - SYMFONY_CONSOLE_VERSION=3.2.*
+  - SYMFONY_CONSOLE_VERSION=3.*
+matrix:
+  include:
+    - php: 7.1
+      env: SYMFONY_CONSOLE_VERSION=4.*
+    - php: 7.2
+      env: SYMFONY_CONSOLE_VERSION=4.*
 
 install:
-  - travis_retry composer install --no-interaction --prefer-source
-
-before_script:
   - composer require symfony/console:${SYMFONY_CONSOLE_VERSION}
```

Comment:
I move the `symfony/console` require before the install to prevent nested depedencies requirements collisions.